SwiftLeeds/Views/My Conference/TalkCell.swift

```diff
@@ -32,14 +34,38 @@ struct TalkCell: View {
 
             VStack(alignment: .leading, spacing: 8) {
                 if let speaker = speaker {
-                    VStack(alignment: .leading, spacing: 4) {
-                        Text(speaker)
-                            .font(.headline.weight(.medium))
+                    HStack {
+                        VStack(alignment: .leading, spacing: 4) {
+                            Text(speaker)
+                                .font(.headline.weight(.medium))
 
-                        if let company = company {
-                            Text(company)
-                                .font(.subheadline.weight(.medium))
-                                .opacity(0.6)
+                            if let company = company {
+                                Text(company)
+                                    .font(.subheadline.weight(.medium))
+                                    .opacity(0.6)
+                            }
+                        }
+
+                        Spacer()
+
+                        if let imageURL = imageURL {
```

Comment:
Nice addition! 🎉

We should make sure this isn't in the way for voiceover users, so let's add accessibilityHidden.

Worth also considering switching this in a content aware stack so it can become a vertical stack instead if they have really big text?